package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-config-formidable",
   "description": "A set of default eslint configurations from formidable",
-  "version": "1.0.0",
+  "version": "0.0.0",
   "authors": "eric.baer@formidable.com, ryan.roemer@formidable.com, kyle.cesmat@formidable.com",
```

Comment:
@baer what is the intended method for making a release? We should probably include documentation on best contributing practices in this repository